### ISSUE TITLE
Any plan for future demo?

### ISSUE THREAD

**Issue by axisdos** [brunnopleffken/addictive-community#6]
Hey man, I've been looking at your progress on Github and it looks like you're doing some amazing stuff. Are you planning to setup a demo link for this software anytime in the future? Looking forward.

**Comment by brunnopleffken** [comment]
Man, a demo link would be really nice for previewing the forum sys before download it... But actually, I don't have any dedicated server for it. But would be nice!! By the way... I'm a little late, I expected to release the first Beta version now in January, but there's still some incomplete features in Admin CP and Private Messenger that I want to fix...

**Comment by axisdos** [comment]
I don't think you will need a dedicated server. A regular shared hosting would be good for now. If you can't afford one I could definitely sponsor.
